### PR TITLE
Chapter 3 Static Pages

### PR DESCRIPTION
**Pull request by Iskandar19**

### CODE REVIEW

**Comment by Iskandar19** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+base_title = "Ruby on Rails Tutorial Sample App"
+page_title.empty? ? base_title : "#{page_title} | #{base_title}"
+end
```
Comment (ghost) [inline app/helpers/application_helper.rb]: fix ident please

File: app/views/static_pages/help.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+<h1>Help</h1>
+<p>
+Get help on the Ruby on Rails Tutorial at the
+<a href="http://www.railstutorial.org/help">Rails Tutorial
+help page</a>.
+To get help on this sample app, see the
+<a href="http://www.railstutorial.org/book"><em>Ruby on Rails
+Tutorial</em>
+book</a>.
```
Comment (ghost) [inline app/views/static_pages/help.html.erb]: fix ident please

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<%= provide :title, "Home" %>
+
+<h1>Sample App</h1>
+<p>
+This is the home page for the
+<a href="http://www.railstutorial.org/">Ruby on
+Rails Tutorial</a>
+sample application.
```
Comment (ghost) [inline app/views/static_pages/home.html.erb]: fix ident please

**Review by ghost — COMMENTED** [review]
cmt

**Comment by Iskandar19** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
+/* universal */
+
+body {
+padding-top: 60px;
```
Comment (ghost) [inline app/assets/stylesheets/custom.scss]: please view this [pull request](https://github.com/lrmi0808/sample_app/pull/2/files ) and fix the ident like it

**Review by ghost — COMMENTED** [review]
cmt

**Comment by Iskandar19** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -10,6 +10,10 @@
...
 
   <body>
-    <%= yield %>
-  </body>
+  <%= render "layouts/header" %>
+   <div class="container">
+     <%= yield %>
+   </div>
+
+</body>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]:
```suggestion
  </body>
```

File: app/views/static_pages/help.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<%= provide :title, "Help" %>
+
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <a href="http://www.railstutorial.org/help">Rails Tutorial
+help page</a>.
```
Comment (ghost) [inline app/views/static_pages/help.html.erb]:
```suggestion
  help page</a>.
```

File: app/views/static_pages/help.html.erb [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  To get help on this sample app, see the
+  <a href="http://www.railstutorial.org/book"><em>Ruby on Rails
+Tutorial</em>
+ book</a>.
```
Comment (ghost) [inline app/views/static_pages/help.html.erb]:
```suggestion
  Tutorial</em>
   book</a>.
```

File: app/views/static_pages/home.html.erb [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Home" %>
+
+<h1>Sample App</h1>
+<p>
+  This is the home page for the
+  <a href="http://www.railstutorial.org/">Ruby on
+Rails Tutorial</a>
+
+  sample application.
```
Comment (ghost) [inline app/views/static_pages/home.html.erb]:
```suggestion
  Rails Tutorial</a>sample application.
```

**Review by ghost — COMMENTED** [review]
cmt

**Comment by Iskandar19** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**